eos-tech-support/eos-diagnostics

```diff
@@ -872,6 +886,10 @@ ARGV.forEach(function(arg) {
         coredumpsFlag = true;
     else if (arg == '-v' || arg == '--verbose')
         verboseFlag = true;
+    else if (arg == '-f' || arg == '--full-journal')
+        // only include the full journal if explicitly requested to limit
+        // exposure of potentially-private information by default
```

Comment:
This comment is misleading. The purpose of not including the full journal by default is purely to avoid producing gigantic files. There is not appreciably less privacy risk from including just two boots.

Reply:
(The commit message is also similarly confused!)